README.md

```diff
@@ -33,3 +33,12 @@ Run in a browser with live-reload:
 ```
 web-ext run --firefox-binary ~/path/to/firefox -s ./extension/
 ```
+
+# Release
+
+1. Bump version in ``package.json``, ``update.json``, and ``extension/manifest.json``
+2. Tag commit ``git tag -a X.Y.Z`` and push it
+3. Create release with changelog on Github
```

Comment:
You shouldn't need to create the tab manually if you're creating a release on github - github should tag it for you.